api-reference/10 UI Components/GridBase/1 Configuration/columnChooser/container.md

```diff
@@ -5,7 +5,102 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a container in which the column chooser should be rendered.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Use this property if you need to change the element in which the column chooser is rendered. For example, if your component is inside the Popup, set this property to popup container.
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $("#{widgetName}").dx{WidgetName}({
+            // ...
+            columnChooser: {
+                // ...
+                container: "#popupContainer"
+            },
+        })
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-{widget-name} ... >
+        <dxo-column-chooser container="#popupContainer" >
+        </dxo-column-chooser>
+    </dx-{widget-name}>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core'; 
```

Comment:
Maybe import the used component here? Similar in the [Column Chooser](https://js.devexpress.com/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Chooser/) help topic (Angular tab):
> import { DxDataGridModule } from "devextreme-angular";